src/uas/UAS.cc

```diff
@@ -147,22 +147,20 @@ UAS::UAS(MAVLinkProtocol* protocol, Vehicle* vehicle) : UASInterface(),
     blockHomePositionChanges(false),
     receivedMode(false),
 
-    // Initialize HIL sensor noise variances to 0.  If user wants corrupted sensor values they will need to set them
-    // Note variances calculated from flight case from this log: http://dash.oznet.ch/view/MRjW8NUNYQSuSZkbn8dEjY
-    // TODO: calibrate stand-still pixhawk variances
-    xacc_var(1.2914f),
-    yacc_var(0.7048f),
-    zacc_var(1.9577f),
-    rollspeed_var(0.8126f),
-    pitchspeed_var(0.6145f),
-    yawspeed_var(0.5852f),
-    xmag_var(0.4786f),
-    ymag_var(0.4566f),
-    zmag_var(0.3333f),
-    abs_pressure_var(1.1604f),
-    diff_pressure_var(1.1604f),
-    pressure_alt_var(1.1604f),
-    temperature_var(1.4290f),
+    // measured pixhawk sensor noise variances

```

Comment:
I would want something in a KISS sense that gets close to real flight conditions. Since we use one model in simulation anyway you can't claim accuracy, so its better to have one "typical" vibration level rather than trying to fit every possible airframe. We want to model only qualitatively, not quantitatively. No need to complicate things we can't model properly anyway.